docs/UserCommands.md

```diff
@@ -0,0 +1,44 @@
+## User Command Reference
+
+
+`@rocket user`
+
+## Options to specify input:
+
+* `-edit`
+    * `-name` somename `-email` email `-pos` yourposition `-github` githublink `-major` yourmajor `-bio` yourbio
+        * edit properties of your Launch Pad profile
+    * ADMIN only option: `-member` MEMBER_ID
```

Comment:
We may want to open this up to team leads as well (or, if we're really ambitious, let team leads do it but only for members of their team... but then again, we may want to leave it so that a team lead can put whoever they want on the team, which kinda torpedoes that plan).